pkg/cmd/api/api.go

```diff
@@ -380,6 +400,10 @@ func apiRun(opts *ApiOptions) error {
 		}
 	}
 
+	if opts.merger != nil {
```

Comment:
Another option here might be to have a `NoOpMerger` that is just a passthrough that is the default for `opts.Merger`. Then we no longer need these `nil` checks. I suppose it's a bit of a `NullObjectPattern`.

I could go either way, I'll leave it up to you. I don't have a strong opinion.